lib/chef/resource/aws_security_group.rb

```diff
@@ -7,11 +7,45 @@ class Chef::Resource::AwsSecurityGroup < Chef::Provisioning::AWSDriver::AWSResou
   actions :create, :delete, :nothing
   default_action :create
 
-  attribute :name,          kind_of: String, name_attribute: true
-  attribute :vpc,           kind_of: [ String, AwsVpc, AWS::EC2::VPC ]
-  attribute :description,   kind_of: String
-  attribute :inbound_rules
-  attribute :outbound_rules
+  attribute :name,           kind_of: String, name_attribute: true
+  attribute :vpc,            kind_of: [ String, AwsVpc, AWS::EC2::VPC ]
```

Comment:
The provider assumes this will always be populated (and it is required in a create call to amazon, unless operating in ec2 classic mode) - can we make it `required: true`?